### PR TITLE
Added test cases for spiders

### PR DESCRIPTION
**Pull request by mohansaimandalapu**
Added test cases for "product_page_url_scraper" and "product_details_scraper" spiders.

### CODE REVIEW

File: tests/closetcoach_crawler/myntra_spider/spiders/test_product_page_urls.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+from scrapy.utils import reactor
+from twisted.internet import asyncioreactor
+
+
+from closetcoach_crawler.myntra_spider.spiders.product_page_urls import (
+    ProductPageUrlScraper,
+)
+from tests.test_data.constants import TEST_PRODUCT_URLS
+
+# Install the AsyncioSelectorReactor and set it as the default reactor for Scrapy
```
Comment (satishjasthi) [inline tests/closetcoach_crawler/myntra_spider/spiders/test_product_page_urls.py:19]: Do we need this to run the spider?, I don't think so

**Review by satishjasthi — CHANGES_REQUESTED** [review]
Thank you for contributing to ClosetCoach @mohansaimandalapu.Overall it looks good to me except few changes I have mentioned in this review. You can update code based on reviews. Then Ill merge your code

File: tests/closetcoach_crawler/myntra_spider/spiders/test_product_page_urls.py (hunk 1 above)
Comment (mohansaimandalapu) [reply tests/closetcoach_crawler/myntra_spider/spiders/test_product_page_urls.py:19]: Yeah, I will do the changes and update the code.

Comment (mohansaimandalapu) [reply tests/closetcoach_crawler/myntra_spider/spiders/test_product_page_urls.py]: @satishjasthi, I have updated the code please check it.

**Review by satishjasthi — APPROVED** [review]
Looks good @mohansaimandalapu. Ill merge this.

**Pull request merged**